types.d.ts

```diff
@@ -4545,7 +4545,7 @@ declare interface InfrastructureLogging {
 	/**
 	 * Log level.
 	 */
-	level?: "none" | "verbose" | "error" | "warn" | "info" | "log";
```

Comment:
running lints for these files produces these re-orderings of unrelated entries as well -- not sure why

Reply:
typescript is a little bit weird here. Afaik this is related to caching of these types in typescript. At least it's not non-deterministic...